packages/fxa-payments-server/src/components/Header/index.tsx

```diff
@@ -1,30 +1,36 @@
 import React from 'react';
-
-import './index.scss';
 import { Profile } from '../../store/types';
+import firefoxLogo from '../../images/firefox-logo-combined.svg';
 
-type HeaderProps = {
+export type HeaderProps = {
   profile?: Profile;
   className?: string;
 };
 
 export const Header = ({ profile, className = 'default' }: HeaderProps) => {
   let profileSection = null;
+
   if (profile) {
     const { avatar, displayName, email } = profile;
     profileSection = (
       <img
-        className="avatar hoisted"
+        className="rounded-full w-8 h-8"
         src={avatar}
         data-testid="avatar"
         alt={displayName || email}
+        title={displayName || email}
       />
     );
   }
 
   return (
-    <header className={`${className} fxa-settings-header`} role="banner">
-      <div className="fxa-logo" data-testid="branding" title="firefox"></div>
+    <header
+      className={`${className} justify-between flex items-center bg-white shadow h-16 fixed left-0 top-0 my-0 mx-auto px-6 py-4 w-full whitespace-nowrap z-10 tablet:h-20 tablet:px-8`}
+      role="banner"
+    >
+      <div data-testid="branding">
+        <img src={firefoxLogo} alt="Firefox logo" />
```

Comment:
Q: Does the `alt` need to be localized?

Reply:
It does indeed. You're a gem! Fixed. Thank you @pdehaan!